__tests__/modules/student/commands/find-one-by-id.command.mock.ts

```diff
@@ -0,0 +1,9 @@
+export const prismaServiceMock = {
+  student: {
+    findUnique: jest.fn(),
+  },
+};
+
+export const findOneByIdFindUniqueExpectedCall = (id: string) => ({
+  where: { user_id: id },
+});
```

Comment:
Como os parâmetros dessa chamada são simples, nem precisa dessa função. Vamo criar ela só quando a chamada esperada for complexa ou tiver muita coisa que não muda